CI e2e admin test_metrics is broken

CI e2e test_metrics is broken: https://github.com/huggingface/dataset-viewer/actions/runs/10319430994/job/28567795517
```
FAILED tests/test_31_admin_metrics.py::test_metrics - AssertionError: queue_jobs_total - queue=dataset-config-names found in {'starlette_requests_total{method="GET",path_template="/admin"}': 3.0, 'starlette_responses_total{method="GET",path_template="/admin",status_code="200"}': 2.0, 'starlette_requests_processing_time_seconds_sum{method="GET",path_template="/admin"}': 0.021941476999927545, 'starlette_requests_processing_time_seconds_bucket{le="0.005",method="GET",path_template="/admin"}': 1.0, 'starlette_requests_processing_time_seconds_bucket{le="0.01",method="GET",path_template="/admin"}': 1.0, 'starlette_requests_processing_time_seconds_bucket{le="0.025",method="GET",path_template="/admin"}': 2.0, 'starlette_requests_processing_time_seconds_bucket{le="0.05",method="GET",path_template="/admin"}': 2.0, 'starlette_requests_processing_time_seconds_bucket{le="0.075",method="GET",path_template="/admin"}': 2.0, 'starlette_requests_processing_time_seconds_bucket{le="0.1",method="GET",path_template="/admin"}': 2.0, 'starlette_requests_processing_time_seconds_bucket{le="0.25",method="GET",path_template="/admin"}': 2.0, 'starlette_requests_processing_time_seconds_bucket{le="0.5",method="GET",path_template="/admin"}': 2.0, 'starlette_requests_processing_time_seconds_bucket{le="0.75",method="GET",path_template="/admin"}': 2.0, 'starlette_requests_processing_time_seconds_bucket{le="1.0",method="GET",path_template="/admin"}': 2.0, 'starlette_requests_processing_time_seconds_bucket{le="2.5",method="GET",path_template="/admin"}': 2.0, 'starlette_requests_processing_time_seconds_bucket{le="5.0",method="GET",path_template="/admin"}': 2.0, 'starlette_requests_processing_time_seconds_bucket{le="7.5",method="GET",path_template="/admin"}': 2.0, 'starlette_requests_processing_time_seconds_bucket{le="10.0",method="GET",path_template="/admin"}': 2.0, 'starlette_requests_processing_time_seconds_bucket{le="+Inf",method="GET",path_template="/admin"}': 2.0, 'starlette_requests_processing_time_seconds_count{method="GET",path_template="/admin"}': 2.0, 'starlette_requests_in_progress{method="GET",path_template="/admin",pid="145"}': 1.0, 'starlette_requests_in_progress{method="GET",path_template="/admin",pid="23"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="dataset-config-names",status="waiting"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="dataset-filetypes",status="waiting"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="dataset-config-names",status="started"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="config-parquet-and-info",status="waiting"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="dataset-parquet",status="waiting"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="dataset-info",status="waiting"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="config-split-names",status="waiting"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="dataset-size",status="waiting"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="dataset-split-names",status="waiting"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="dataset-is-valid",status="waiting"}': 1.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="dataset-opt-in-out-urls-count",status="waiting"}': 1.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="dataset-duckdb-index-size",status="waiting"}': 1.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="dataset-filetypes",status="started"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="dataset-modalities",status="waiting"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="dataset-duckdb-index-size",status="started"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="dataset-info",status="started"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="dataset-compatible-libraries",status="waiting"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="dataset-croissant-crumbs",status="waiting"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="dataset-is-valid",status="started"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="dataset-hub-cache",status="waiting"}': 1.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="dataset-opt-in-out-urls-count",status="started"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="dataset-parquet",status="started"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="dataset-size",status="started"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="dataset-split-names",status="started"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="dataset-presidio-entities-count",status="waiting"}': 1.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="config-split-names",status="started"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="split-first-rows",status="waiting"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="config-is-valid",status="waiting"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="config-opt-in-out-urls-count",status="waiting"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="config-duckdb-index-size",status="waiting"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="config-parquet-and-info",status="started"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="config-parquet",status="waiting"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="config-info",status="waiting"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="config-size",status="waiting"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="dataset-modalities",status="started"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="dataset-compatible-libraries",status="started"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="dataset-croissant-crumbs",status="started"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="dataset-hub-cache",status="started"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="dataset-presidio-entities-count",status="started"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="config-duckdb-index-size",status="started"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="config-is-valid",status="started"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="config-opt-in-out-urls-count",status="started"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="split-first-rows",status="started"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="split-is-valid",status="waiting"}': 1.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="split-image-url-columns",status="waiting"}': 1.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="config-info",status="started"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="config-parquet",status="started"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="config-parquet-metadata",status="waiting"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="config-size",status="started"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="split-is-valid",status="started"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="split-image-url-columns",status="started"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="split-opt-in-out-urls-scan",status="waiting"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="config-parquet-metadata",status="started"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="split-descriptive-statistics",status="waiting"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="split-presidio-scan",status="waiting"}': 1.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="split-duckdb-index",status="waiting"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="split-opt-in-out-urls-scan",status="started"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="split-opt-in-out-urls-count",status="waiting"}': 1.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="split-descriptive-statistics",status="started"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="split-duckdb-index",status="started"}': 1.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="split-presidio-scan",status="started"}': 0.0, 'queue_jobs_total{dataset_status="normal",pid="145",queue="split-opt-in-out-urls-count",status="started"}': 0.0, 'worker_size_jobs_count{pid="145",worker_size="medium"}': 1.0, 'worker_size_jobs_count{pid="145",worker_size="light"}': 8.0, 'responses_in_cache_total{error_code="None",http_status="200",kind="dataset-config-names",pid="145"}': 7.0, 'responses_in_cache_total{error_code="None",http_status="200",kind="dataset-filetypes",pid="145"}': 7.0, 'responses_in_cache_total{error_code="None",http_status="200",kind="dataset-duckdb-index-size",pid="145"}': 7.0, 'responses_in_cache_total{error_code="None",http_status="200",kind="dataset-info",pid="145"}': 7.0, 'responses_in_cache_total{error_code="None",http_status="200",kind="dataset-is-valid",pid="145"}': 7.0, 'responses_in_cache_total{error_code="None",http_status="200",kind="dataset-opt-in-out-urls-count",pid="145"}': 7.0, 'responses_in_cache_total{error_code="None",http_status="200",kind="dataset-parquet",pid="145"}': 7.0, 'responses_in_cache_total{error_code="None",http_status="200",kind="dataset-size",pid="145"}': 7.0, 'responses_in_cache_total{error_code="None",http_status="200",kind="dataset-split-names",pid="145"}': 7.0, 'responses_in_cache_total{error_code="None",http_status="200",kind="config-split-names",pid="145"}': 7.0, 'responses_in_cache_total{error_code="None",http_status="200",kind="config-parquet-and-info",pid="145"}': 7.0, 'responses_in_cache_total{error_code="None",http_status="200",kind="dataset-modalities",pid="145"}': 7.0, 'responses_in_cache_total{error_code="None",http_status="200",kind="dataset-compatible-libraries",pid="145"}': 3.0, 'responses_in_cache_total{error_code="None",http_status="200",kind="dataset-croissant-crumbs",pid="145"}': 7.0, 'responses_in_cache_total{error_code="None",http_status="200",kind="dataset-hub-cache",pid="145"}': 7.0, 'responses_in_cache_total{error_code="None",http_status="200",kind="dataset-presidio-entities-count",pid="145"}': 7.0, 'responses_in_cache_total{error_code="None",http_status="200",kind="config-duckdb-index-size",pid="145"}': 7.0, 'responses_in_cache_total{error_code="None",http_status="200",kind="config-is-valid",pid="145"}': 7.0, 'responses_in_cache_total{error_code="None",http_status="200",kind="config-opt-in-out-urls-count",pid="145"}': 7.0, 'responses_in_cache_total{error_code="None",http_status="200",kind="split-first-rows",pid="145"}': 3.0, 'responses_in_cache_total{error_code="None",http_status="200",kind="config-info",pid="145"}': 7.0, 'responses_in_cache_total{error_code="None",http_status="200",kind="config-parquet",pid="145"}': 7.0, 'responses_in_cache_total{error_code="None",http_status="200",kind="config-size",pid="145"}': 7.0, 'responses_in_cache_total{error_code="None",http_status="200",kind="split-is-valid",pid="145"}': 7.0, 'responses_in_cache_total{error_code="None",http_status="200",kind="split-image-url-columns",pid="145"}': 3.0, 'responses_in_cache_total{error_code="None",http_status="200",kind="config-parquet-metadata",pid="145"}': 5.0, 'responses_in_cache_total{error_code="MissingSpawningTokenError",http_status="500",kind="split-opt-in-out-urls-scan",pid="145"}': 7.0, 'responses_in_cache_total{error_code="None",http_status="200",kind="split-descriptive-statistics",pid="145"}': 3.0, 'responses_in_cache_total{error_code="None",http_status="200",kind="split-duckdb-index",pid="145"}': 2.0, 'responses_in_cache_total{error_code="None",http_status="200",kind="split-presidio-scan",pid="145"}': 2.0, 'responses_in_cache_total{error_code="MissingSpawningTokenError",http_status="500",kind="split-opt-in-out-urls-count",pid="145"}': 6.0, 'responses_in_cache_total{error_code="FileSystemError",http_status="500",kind="config-parquet-metadata",pid="145"}': 2.0, 'responses_in_cache_total{error_code="UnexpectedError",http_status="500",kind="dataset-compatible-libraries",pid="145"}': 4.0, 'responses_in_cache_total{error_code="InfoError",http_status="500",kind="split-first-rows",pid="145"}': 2.0, 'responses_in_cache_total{error_code="FileSystemError",http_status="500",kind="split-descriptive-statistics",pid="145"}': 2.0, 'responses_in_cache_total{error_code="FileSystemError",http_status="500",kind="split-duckdb-index",pid="145"}': 2.0, 'responses_in_cache_total{error_code="PresidioScanNotEnabledForThisDataset",http_status="501",kind="split-presidio-scan",pid="145"}': 3.0, 'responses_in_cache_total{error_code="InfoError",http_status="500",kind="split-image-url-columns",pid="145"}': 2.0, 'responses_in_cache_total{error_code="UnexpectedError",http_status="500",kind="split-first-rows",pid="145"}': 2.0, 'responses_in_cache_total{error_code="UnexpectedError",http_status="500",kind="split-descriptive-statistics",pid="145"}': 2.0, 'responses_in_cache_total{error_code="UnexpectedError",http_status="500",kind="split-duckdb-index",pid="145"}': 2.0, 'responses_in_cache_total{error_code="UnexpectedError",http_status="500",kind="split-image-url-columns",pid="145"}': 2.0, 'responses_in_cache_total{error_code="StreamingRowsError",http_status="500",kind="split-first-rows",pid="145"}': 0.0, 'responses_in_cache_total{error_code="StreamingRowsError",http_status="500",kind="split-image-url-columns",pid="145"}': 0.0, 'responses_in_cache_total{error_code="NormalRowsError",http_status="500",kind="split-presidio-scan",pid="145"}': 1.0, 'parquet_metadata_disk_usage{pid="145",type="total"}': 77851254784.0, 'parquet_metadata_disk_usage{pid="145",type="used"}': 76050628608.0, 'parquet_metadata_disk_usage{pid="145",type="free"}': 1783848960.0, 'parquet_metadata_disk_usage{pid="145",type="percent"}': 97.7}
assert False
 +  where False = has_metric(name='queue_jobs_total', labels={'pid': '[0-9]*', 'queue': 'dataset-config-names', 'status': 'started'}, metric_names={'parquet_metadata_disk_usage{pid="145",type="free"}', 'parquet_metadata_disk_usage{pid="145",type="percent"}', 'parquet_metadata_disk_usage{pid="145",type="total"}', 'parquet_metadata_disk_usage{pid="145",type="used"}', 'queue_jobs_total{dataset_status="normal",pid="145",queue="config-duckdb-index-size",status="started"}', 'queue_jobs_total{dataset_status="normal",pid="145",queue="config-duckdb-index-size",status="waiting"}', ...})
```

I think this failure was introduced after merging: https://github.com/huggingface/dataset-viewer/actions/runs/10182877247/job/28845665481
- #3008

--- a/e2e/tests/test_21_api_metrics.py
+++ b/e2e/tests/test_21_api_metrics.py
@@ -31,7 +31,7 @@ def test_metrics() -> None:
         # these metrics are only available in the admin API
         assert not has_metric(
             name="queue_jobs_total",
-            labels={"pid": "[0-9]*", "queue": endpoint, "status": "started"},
+            labels={"dataset_status": "normal", "pid": "[0-9]*", "queue": endpoint, "status": "started"},
             metric_names=metric_names,
         ), f"queue_jobs_total - endpoint={endpoint} found in {metrics}"
         assert not has_metric(

--- a/e2e/tests/test_31_admin_metrics.py
+++ b/e2e/tests/test_31_admin_metrics.py
@@ -25,7 +25,7 @@ def test_metrics() -> None:
     metric_names = set(metrics.keys())
 
     # the queue metrics are computed each time a job is created and processed
-    # they should exists at least for some of jobs types
+    # they should exist at least for some of jobs types
     for queue in ["dataset-config-names", "split-first-rows", "dataset-parquet"]:
         # eg. 'queue_jobs_total{pid="10",queue="split-first-rows",status="started"}'
         assert has_metric(
@@ -35,7 +35,7 @@ def test_metrics() -> None:
         ), f"queue_jobs_total - queue={queue} found in {metrics}"
 
     # the queue metrics are computed each time a job is created and processed
-    # they should exists at least for some of jobs types
+    # they should exist at least for some of jobs types
     for worker_size in [
         "light",
         "medium",
@@ -48,7 +48,7 @@ def test_metrics() -> None:
         ), f"worker_size_jobs_count - worker_size={worker_size} found in {metrics}"
 
     # the cache metrics are computed each time a job is processed
-    # they should exists at least for some of cache kinds
+    # they should exist at least for some of cache kinds
     for cache_kind in ["dataset-config-names", "split-first-rows", "dataset-parquet"]:
         # cache should have been filled by the previous tests
         # eg. 'responses_in_cache_total{error_code="None",http_status="200",path="dataset-config-names",pid="10"}'

--- a/e2e/tests/test_31_admin_metrics.py
+++ b/e2e/tests/test_31_admin_metrics.py
@@ -30,7 +30,7 @@ def test_metrics() -> None:
         # eg. 'queue_jobs_total{pid="10",queue="split-first-rows",status="started"}'
         assert has_metric(
             name="queue_jobs_total",
-            labels={"pid": "[0-9]*", "queue": queue, "status": "started"},
+            labels={"dataset_status": "normal", "pid": "[0-9]*", "queue": queue, "status": "started"},
             metric_names=metric_names,
         ), f"queue_jobs_total - queue={queue} found in {metrics}"
 

--- a/e2e/tests/test_41_rows_metrics.py
+++ b/e2e/tests/test_41_rows_metrics.py
@@ -31,7 +31,7 @@ def test_metrics() -> None:
         # these metrics are only available in the admin API
         assert not has_metric(
             name="queue_jobs_total",
-            labels={"pid": "[0-9]*", "queue": endpoint, "status": "started"},
+            labels={"dataset_status": "normal", "pid": "[0-9]*", "queue": endpoint, "status": "started"},
             metric_names=metric_names,
         ), f"queue_jobs_total - endpoint={endpoint} found in {metrics}"
         assert not has_metric(

--- a/e2e/tests/test_51_search_metrics.py
+++ b/e2e/tests/test_51_search_metrics.py
@@ -31,7 +31,7 @@ def test_metrics() -> None:
         # these metrics are only available in the admin API
         assert not has_metric(
             name="queue_jobs_total",
-            labels={"pid": "[0-9]*", "queue": endpoint, "status": "started"},
+            labels={"dataset_status": "normal", "pid": "[0-9]*", "queue": endpoint, "status": "started"},
             metric_names=metric_names,
         ), f"queue_jobs_total - endpoint={endpoint} found in {metrics}"
         assert not has_metric(

